src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborReader.Map.cs

```diff
@@ -5,16 +5,28 @@
 #nullable enable
 using System.Collections.Generic;
 using System.Diagnostics;
-using System.Diagnostics.CodeAnalysis;
-using System.Threading;
 
 namespace System.Formats.Cbor
 {
     public partial class CborReader
     {
         private KeyEncodingComparer? _keyEncodingComparer;
-        private Stack<HashSet<(int Offset, int Length)>>? _pooledKeyEncodingRangeSets;
-
+        private Stack<HashSet<(int Offset, int Length)>>? _pooledKeyEncodingRangeAllocations;
+
+        /// <summary>
+        ///   Reads the next data item as the start of a map (major type 5)
```

Comment:
I think the rest of your summaries have had periods, but this one was missing it. Having the periods is correct (https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Method)

```suggestion
        ///   Reads the next data item as the start of a map (major type 5).
```